index.html

```diff
@@ -42,7 +36,7 @@ <h3>I'am a Web Developer.</h3>
 	<!----About section start---->
 	<section class="about">
 		<div class="main">
-			<img src="img/main-img.png">
+			<img src="img/main-img.png" alt="">
```

Comment:
I suggest that you include an alternative for the image because sometimes the image may not display.